### DISCUSSION TITLE
如何设置某一颜色为默认主题色？

### DISCUSSION THREAD

**Discussion by tenngoxars** [vuepress-theme-hope/vuepress-theme-hope#549] [Q&A] [answered]
通过`themeColor`设置了可选的颜色，但默认进入主页后的颜色仍然是 Vue 绿，请问默认的颜色应该在何处设置？

**Comment by Mister-Hope** [comment, marked as answer]
在palette.styl 设置 $accentColor，设置方式同 VuePress 默认主题
https://vuepress-theme.mrhope.site/zh/config/stylus/#%E9%A2%9C%E8%89%B2
